.github/workflows/release.yml

```diff
@@ -62,14 +62,14 @@ jobs:
       - name: Semantic Release runs for draft release
         if: ${{github.event.inputs.draft-release == 'true'}}
         id: semantic-release-draft
-        run: npx semantic-release -t \${version}
+        run: DEBUG=semantic-release:* $(npm bin)/semantic-release  -t \${version}
```

Comment:
Can you add more context why this changes are needed and how it solves the issue?

Reply:
I don't know why doesn't work with npx command. This is the issue https://github.com/semantic-release/semantic-release/issues/2297 which I raised on semantic-release repo.

Reply:
Okay, So we're just adding debug logs for it? 

Also - Should it be `${npm bin}` instead of `$(npm bin)`?

Reply:
https://stackoverflow.com/questions/27472540/difference-between-and-in-bash/27472808
It should $(npm bin) and yes we added debug for better logs